cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1840,10 +1843,10 @@ subroutine input_data
                   tmpstr2 = ' : no wave data provided, no wave-ice interactions'
                elseif (trim(wave_spec_type) == 'profile') then
                   tmpstr2 = ' : use fixed dummy wave spectrum for testing'
-               elseif (trim(wave_spec_type) == 'constant') then
+               elseif (trim(wave_spec_type) == 'constant_file') then
                   tmpstr2 = ' : constant wave spectrum data file provided for testing'
-               elseif (trim(wave_spec_type) == 'random') then
```

Comment:
I do, but I moved that switch to be included in the wave_solver flag. 

'wave_spec_type' determines the type of wave data we're reading in:

wave_spec_type = 'none' - No wave data, no wave-ice interactions are simulated
wave_spec_type = 'profile' - Set wave spectrum to dummy profile at all locations
wave_spec_type = 'constant_file' - Read in file for wave spectra with one timestep, i.e. wave spectra are constant in time but not in space. You might use this option for coupling with an actual wave model 
wave_spec_type = 'forcing_file'  - Read in forcing file with 6 hourly data, i.e. wave spectra vary in time and space. I can remove this if you prefer, I just thought it might be helpful.

'wave_solver' determines the way the wave fracture is computed. It doesn't matter what this value is if wave_spec_type = 'none'. (Maybe we also want to set  'wave_solver='none' in the default namelist?)

wave_solver = ‘ml' - use machine learning emulation of wave fracture. This does not call random numbers. Will be BFB for itself, will not BFB with the previous version of the code (with wave_spec_type=constant), but code should be BFB with previous version when wave_spec_type = 'none' (no wave ice interactions)

wave_solver = ‘std-1iter’ - this is what we called ‘constant’ in the previous version. This should be BFB with itself. Should be BFB with wave_spec_type=‘constant’ in previous code version. I named it like this because in this case we only iterate the fracture code once, and it doesn't converge.

wave_solver = ‘std-conv’ - this is what we called ‘random’ in the previous version. This has a random number call so will not be BFB. We iterate over random generations of the sea surface height field until the wave fracture scheme reaches convergence, so it's very expensive.

Does this make sense?